contrib/seeds/makeseeds.py

```diff
@@ -197,7 +198,7 @@ def main():
     print(f'{ip_stats(ips):s} Skip entries with invalid address', file=sys.stderr)
     # Skip duplicates (in case multiple seeds files were concatenated)
     ips = dedup(ips)
-    print(f'{ip_stats(ips):s} After removing duplicates', file=sys.stderr)
+    print(f'{ip_stats(ips):s} Remove duplicates', file=sys.stderr)
```

Comment:
I think this can also just be left as-is. "After removing duplicates" certainly fits the tone/amongst the other sentences better. Not really sure why it needs changing.

Reply:
I don't really mind, I just don't see why there's multiple unrelated changes in this PR (manual copyright header bumping, rewriting log output etc), when it could just be a straight-forward seed update.